ci/build_linux.sh

```diff
@@ -28,7 +28,7 @@ checkError $? "cmake build failed for pxcore or rtcore" "Compilation error" "Che
 
 else
 echo "***************************** Generating config files ****"
-cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON .. 1>>$BUILDLOGS;
+cmake -DBUILD_PX_TESTS=ON -DBUILD_PXSCENE_STATIC_LIB=ON -DENABLE_DEBUG_METRICS=ON .. 1>>$BUILDLOGS;
```

Comment:
Please rename "-DENABLE_DEBUG_METRICS=ON" to "-DBUILD_DEBUG_METRICS=ON".  Please see comment in CMakeLists.txt for more details.